### ISSUE TITLE
What is the status of this script? How can it be used?

### ISSUE THREAD

**Issue by rbdl1** [drewipson/PyRobinhood#1]

**Comment by drewipson** [comment]
@rbdl1 
The current status of the script is to allow the user to view your portfolio information. You can login and see a top-sheet overview of your portfolio. The data offered goes in this order: Ticker, Current Price per stock, average purchase price per stock, quantity, total equity, and then your return on investment as a percentage.

**Comment by rbdl1** [comment]
Yes, but I still don't understand how one can actually run it? Maybe a README with instructions for running the program would be helpful.
